examples/config.json

```diff
@@ -184,6 +200,22 @@
         "gpus"     : 6
     },
 
+    "ornl.frontier"  : {
+        "project"  : "chm155_003",
+        "queue"    : "batch",
+        "schema"   : "local",
+        "cores"    : 10,
+        "gpus"     : 0
+    },
+
+    "ornl.frontier_flux"  : {
+        "project"  : "chm155_003",
+        "queue"    : "batch",
+        "schema"   : "local",
+        "cores"    : 10,
+        "gpus"     : 0
+    },
```

Comment:
duplications

Reply:
`frontier` vs. `frontier_flux`?

Reply:
here just two times records of both `frontier` and `frontier_flux`

Reply:
Thanks, fixed now.